src/NUnitFramework/tests/Constraints/AnyOfConstraintTests.cs

```diff
@@ -44,6 +44,27 @@ public void ItemIsPresent_IgnoreCaseWhiteSpace()
             Assert.That(anyOf.ApplyTo("A B").Status, Is.EqualTo(ConstraintStatus.Success));
         }
 
+        [Test]
+        public void ItemIsPresent_NormalizeLineEndings()
+        {
+            var anyOf = new AnyOfConstraint(new[] { "a", "B", "a\nb\r" }).NormalizeLineEndings;
+            Assert.That(anyOf.ApplyTo("a\rb\r\n").Status, Is.EqualTo(ConstraintStatus.Success));
+        }
+
+        [Test]
+        public void ItemIsPresent_IgnoreCaseNormalizeLineEndings()
+        {
+            var anyOf = new AnyOfConstraint(new[] { "a", "B", "A\nb\r" }).IgnoreCase.NormalizeLineEndings;
+            Assert.That(anyOf.ApplyTo("a\rB\r\n").Status, Is.EqualTo(ConstraintStatus.Success));
+        }
+
+        [Test]
+        public void ItemIsPresent_IgnoreWhiteSpaceNormalizeLineEndings()
+        {
+            var anyOf = new AnyOfConstraint(new[] { "a", "B", "a b" }).IgnoreWhiteSpace;
+            Assert.That(anyOf.ApplyTo("ab").Status, Is.EqualTo(ConstraintStatus.Success));
+        }
+
```

Comment:
You didn't add the `NormalizeLineEndings` modifiers to proof that it doesn't break things.
Even though there are no line endings when white space is ignored.